### ISSUE TITLE
save/export render?

### ISSUE THREAD

**Issue by ssh4net** [houmain/gpupad#3]
Sorry for the question, but documentation is pretty limited.

How is it possible to save/export rendered image[s] to a file?

Thank you.

**Comment by houmain** [comment]
Sure, simply focus the render output by clicking it and press the Save button.
Thanks for your interest!

**Comment by ssh4net** [comment]
Oh, you right. Just checked Quad example and it saving PNG output.
So probably I missed something in my setup.

Amazing tool! Thank you for development.

**Comment by ssh4net** [comment]
@houmain I'm sorry.
Looks like a bug in this module.

When everything work: when user make new script and first time save Target texture. For example if I'll open any example and save render it save.

When it stop working:
1) when user try to change Target texture resolution or properties. GPUpad sometime completely ignore this.
Only way it create new texture, define new properties set it as a target and some chance that it will work.
2) when user will try to save as new texture file. In that case project will do something like it saving but no file. After that if user will try to save or save all, GPUpad will pop up save as dialogue show ExistName(1).ext but no file are exist. Rename it back without (1) will pop up dialogue after press save button on that dialogue.

I also tried to create new texture and use it as a target without deleting old texture and that also never worked.

Basically for this moment only usable way create new project for any changes :(

**Comment by houmain** [comment]
Hi, thanks

**Comment by houmain** [comment]
Thanks for your feedback!
I think I could reproduce and fix both problems. 
Are you building from source or do you need a release to try it out?

**Comment by ssh4net** [comment]
@houmain Anything is good for you.

I use Visual Studio 2022 with VCPKG for my own projects, and usually should not have issues to build binaries myself.

Btw, did not found any social network accounts in your profile. But will be glad to follow you somewhere.
Also how about enable discussions for that project. Because some questions or maybe feedback about tool are not well suitable for issues.

**Comment by houmain** [comment]
I just released a version 1.29 and enabled the Discussion feature as you suggested.

**Comment by ssh4net** [comment]
Sorry, still cant make save working.
For sample projects now changes for target texture work well. But saving for own project that have same structure not working.
Maybe you can check project file in attachment?
i use RGB 16bit outpit.
and also have "debug" code in shader same as in one of examples with mouse xoordinates.
[Distort_Clean.zip](https://github.com/houmain/gpupad/files/10353373/Distort_Clean.zip)

**Comment by houmain** [comment]
Hi, I tried your session file.
Saving RGB 16bit Textures to PNG indeed does not work currently. I will have a look at it.
But saving RGBA 16bit to PNG or saving RGB 16bit to KTX should already work.

**Comment by ssh4net** [comment]
Oh, can you also check tiff please?
Actually it even more usable because much faster on save than PNG on big 4-8K files.
